assets/source/setup-guide/app/steps/ClaimWebsite.test.js

```diff
@@ -32,9 +33,17 @@ import { fireEvent, render } from '@testing-library/react';
  */
 import ClaimWebsite from './ClaimWebsite';
```

Comment:
💅  

Could we remove as well

`jest.mock( '@woocommerce/tracks'`

I guess I forgot myself to remove it when we implemented this in `test/dependencies` as discussed.